projects/table-builder/src/lib/components/column-builder/column-builder.component.ts

```diff
@@ -75,6 +79,16 @@ export class ColumnBuilderComponent implements OnInit {
     }
   }
 
+  mapWidth = (currentAndPrevious : [number, number]) => {
+    const previousWidth = currentAndPrevious[0];
+    const currentWidth = currentAndPrevious[1]
+    if(currentWidth >= 0){
```

Comment:
use a method name or variable names that describe the why of why we're using this logic.